src/west/app/project.py

```diff
@@ -729,9 +729,8 @@ def rev_info(rev):
         head_info = rev_info('HEAD')
         # If manifest-rev is missing, we already failed earlier.
         manifest_rev_info = rev_info('manifest-rev')
-        if head_info != manifest_rev_info:
-            self.small_banner(f'manifest-rev: {manifest_rev_info}')
-            self.inf(f'            HEAD: {head_info}')
+        self.small_banner(f'manifest-rev: {manifest_rev_info}')
+        self.inf(f'            HEAD: {head_info}')
```

Comment:
I'm not against this and if it makes your life easier, OK. But please update the comment at the top of the function (remove ", but only if they are different.")

Reply:
Done.